common/i18n/resources/en.js

```diff
@@ -408,6 +408,7 @@ The FRA team
   },
 
   nationalDataPoint: {
+    clickOnNDP: "Click on the year to access original data",
```

Comment:
in French this is : Cliquez sur l'année pour accéder aux données originales
in Spanish: Haga clic en el año para acceder a los datos originales
In Russian check your email.. thanks!